docs/blog/newsletter-november-2018.rst

```diff
@@ -0,0 +1,118 @@
+  - Letting Go of the Words (Janice Redish)
+  - Modern Technical Writing (Andrew Etter)
+  - The Product is Docs (Christopher Gales & the Splunk Documentation Team)
+- **General interest**
+
+  - Diffusion of Innovations (Everett M. Rodgers)
+  - How Learning Works (Susan A. Ambrose, Michael W. Bridges, and Michelle DiPietro)
+  - Universal Principles of Design, Revised and Updated (William Lidwell, Kritrina Holden, Jill Butler)
+  - The Wisdom of Crowds (James Surowiecki)
+- **Reference and style**
+
+  - Elements of Style (Strunk & White)
+  - Microsoft Manual of Style
+
+------------------------
+Promoting plain language
+------------------------
+
+Recently, a new Slack member 🎉 used their first post in #general to ask about everyone's experiences with encouraging plain language at work.
+
+First off, most participants agreed that distributing a style guide isn't enough, even with support from management. It can help to include justifications for the rules in your style guide - not only can it win colleagues over, it also helps people remember them. One documentarian suggested pairing every style guide rule with a second line explaining the rationale for it.
+
+Another idea was to target a few issues at a time. For example, you might start by getting everyone on board with "using" rather than "utilizing," then move on to eliminating passive voice.
+
+If you're the lone writer, you might have luck with transitioning to plain language without announcing it and addressing concerns as they come up.
+
+More ideas for winning colleagues over included:
+
+- Developing internal content using plain language to demonstrate its usefulness.
+- Explaining that plain language improves translation accuracy and reduces translation costs, if that's relevant to your content.
+
+If you need help wrangling tech docs into plain language, try looking to your marketing. Marketing materials tend to use less technical language, and they might give you ideas for simpler ways to phrase things in your tech docs.
+
+No matter what, adopting and implementing plain language seems to be a work in progress: almost everyone mentioned that they are looking for opportunities to improve.
+
+---------------------------
+How to hire a documentarian
+---------------------------
+
+How to hire is a topic of perennial interest. So fortunately, there's a great new addition to the Write the Docs website: the `Guide to Hiring and Getting Hired <https://www.writethedocs.org/hiring-guide/>`_.
+
+It came from the Writing Day at the Prague conference, when a group of documentarians got together to collect the questions that they use in interviews, and the questions they like to ask when they're *being* interviewed. All of this knowledge is now written up in the guide, aimed at helping out candidates and hiring managers alike.
+
+Thanks to `Kristy James <https://github.com/kristyj>`_ for running the session and putting the guide together, and to everyone who contributed words and ideas!
+
```

Comment:
Remove comma after "together"
...guide together and to everyone...

Reply:
I'd prefer to keep it in - feels more natural to me